packages/react/src/components/Authenticator/ForceNewPassword/ForceNewPassword.tsx

```diff
@@ -41,11 +41,8 @@ export const ForceNewPassword = ({
           <Heading level={3}>{translate('Change Password')}</Heading>
 
           <FormFields />
-          {error && (
-            <Text className="forceNewPasswordErrorText" variation="error">
-              {error}
-            </Text>
-          )}
+
+          <RemoteErrorMessage />
```

Comment:
This should be using `RemoteErrorMessage` so it looks and behaves like our other error messages in other components.

Reply:
Nice and clean!